react/.eslintrc.json

```diff
@@ -16,16 +21,14 @@
 
     "react/jsx-indent-props": [2, 2],
     "react/jsx-handler-names": [2, { "eventHandlerPrefix": "on", "eventHandlerPropPrefix": "on" }],
-    "react/jsx-boolean-value": [2, "never"],
     "react/jsx-closing-bracket-location": [2, { "nonEmpty": "line-aligned", "selfClosing": "tag-aligned" }],
     "react/jsx-curly-spacing": [2, "never"],
     "react/jsx-equals-spacing": [2, "never"],
     "react/jsx-first-prop-new-line": [2, "multiline"],
     "react/jsx-max-props-per-line": [2, { "maximum": 3 }],
     "react/jsx-pascal-case": [2, { "allowAllCaps": true }],
-    "react/jsx-space-before-closing": [2, "always"],
 
-    "react/jsx-tag-spacing": 2,
+    "react/jsx-tag-spacing": ["error", { "beforeSelfClosing": "always" }],
```

Comment:
Why has this changed?

Reply:
Also, why not default config?

Reply:
Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
The react/jsx-space-before-closing rule is deprecated. Please use the react/jsx-tag-spacing rule with the "beforeSelfClosing" option instead.

Reply:
The react/jsx-space-before-closing rule is deprecated. Please use the react/jsx-tag-spacing rule with the "beforeSelfClosing" option instead.

Reply:
Not sure what we want here.

Reply:
`"beforeSelfClosing": "always"` is the default. So you can remove the override.

Reply:
Removed, ta

Reply:
Plz

```suggestion
    "react/jsx-tag-spacing": 2,
```